messages/create.json

```diff
@@ -6,6 +6,8 @@
     "$ sfdx force:source:manifest:create --sourcepath force-app --manifestname myNewManifest"
```

Comment:
Can we add an example of generating a manifest from an org? Something like:

$ sfdx force:source:manifest:create --fromorg test@myorg.com --includepackages managed

Reply:
I've added the example.